### ISSUE TITLE
Filter coal and the Respirator from Misty World not lasting long enough for the task

### ISSUE THREAD

**Issue by KingofThemyscira** [davqvist/Exoria#251]
Filter coal is only lasting about 25 seconds maybe 30 in the fog of the misty world. I was watching @ChosenArchitect and his filter coal was lasting much longer even thought it's the same coal. Maybe there have been changing to the coal since then? Even with a decent handful of the filter coal it's not possible to build a mine shaft to the lower Y levels of misty world. I'm kinda stuck since I need the Hard Foggy Stone. Is there any other way to get this done or protection from the fog at this point

I suggest significantly increasing the lifespan of the filter coal.  Please help, can't progress!

**Comment by davqvist** [comment]
I wish I knew what is going on. Several people reported of different lifespans of the Filter coal. I suspect that the rate of pollution depends on where you are, but don't quote me on that. Try going down at a different spot maybe. Please report back because I like to understand what is going on as well.

**Comment by KingofThemyscira** [comment]
@davqvist Ok so I tried in my normal world and I made a test world.  I tried at night, daytime, many different Y levels, as many different biomes that I could find and the results have been the same every test, the filter coal is only lasting for roughly 45-48 seconds.

**Comment by davqvist** [comment]
I watched a stream today and the Filter Coal lasted easily 10 minutes, I will ask the dev.

**Comment by KingofThemyscira** [comment]
@davqvist  Another example is the Youtuber SystemCollapse.  You can see here https://www.youtube.com/watch?v=YqMeGwAbtFY&list=PLJkfWFKMxbFjEUD6Ic_lYpkcpLzY3Hl4z&index=15 
8 days ago he hit this exact point, ran into the same issue and hasn't published a video on the pack since then.  Seems he is stuck just like me.  

Some random person in the comments of this video says you need to Update Misty world to c_05.  The near-insta-death doesn't happen anymore.  I'm assuming that is something you would do?

**Comment by KingofThemyscira** [comment]
I'm on curseforge right now and the current Misty world release is c_06, Exoria is running c_04.  I can just switch those out in my own mod file right? I haven't done much with this stuff.

**Comment by KingofThemyscira** [comment]
@davqvist Ok confirmed, I manually updated Misty World to its current release c_06 and the coal last way longer.  Thank you so much for you quick responses.

**Comment by davqvist** [comment]
I did not update intentionally to new versions as it adds world gen that I need to disable (Lapis Ore), Be aware of that. But I figured it out, it depends on how far away you are away from the center of the planet. There is a compass like device that shows you the direction to the center. In the center the Filter Coal lasts much less time. I will add a quest and a crafting recipe to 1.1.0 of the pack
